src/CMakeLists.txt

```diff
@@ -29,7 +30,8 @@ set(RAJA_PERFSUITE_EXECUTABLE_DEPENDS
     polybench
     stream
     stream-kokkos
-    algorithm)
+    algorithm
+    comm)
```

Comment:
I've added comm to the list of dependencies here but libcomm.a is not ending up on the link line of the final executable. Does anyone know what I'm missing?

Reply:
Thanks for finding the issue @rhornung67